Taxi/Taxi/Presenter/ChatRoomView.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  ChatRoomView.swift
+//  Taxi
+//
+//  Created by 이윤영 on 2022/06/13.
+//
+
+import SwiftUI
+
+struct ChatRoomView: View {
+    @State var party: TaxiParty
```

Comment:
이전 뷰에서 데이터를 받는 경우는 Binding으로 받거나 이 화면에서 수정될 일이 없으면 그냥 변수로 받으면 되지 않나요?
이전 뷰에서도 @State로 되어 있고 여기서도 @State로 되어 있으면 Source of Truth인가 뭐시기에 위배되는 것 아닌가요?
```suggestion
    @Binding var party: TaxiParty
```
<br/>

또는 계속 이전 뷰에 의존하게 된다면 상수로만 받아도 될 것 같습니다만, 이건 나중에 정보를 어떻게 불러오느냐에 따라 달라질 것 같네요.
```suggestion
    let party: TaxiParty
```

Reply:
프로토타이핑 화면상 수정될 일이 없을 것 같아 let으로 변경했습니다. 감사합니다

Reply:
근데 여기도 접근지정자를 넣어줘야 하는지는 잘 모르겠네요. @HoJongPARK 도움!

Reply:
생성자를 직접 구현하면 private 으로 선언해도 되는데, 생성자를 구현하지 않은 채로 private 으로 바꾸면 에러 뜰걸요